streams-runtimes/streams-runtime-threaded/src/main/java/org/apache/streams/threaded/controller/ThreadingController.java

```diff
@@ -54,9 +55,9 @@
 
     private static final Integer NUM_PROCESSORS = Runtime.getRuntime().availableProcessors();
 
-    private static final ThreadingController INSTANCE_LOW_PRIORITY = new ThreadingController("Apache Streams [low]", NUM_PROCESSORS, NUM_PROCESSORS * 2, Thread.NORM_PRIORITY - 2);
-    private static final ThreadingController INSTANCE = new ThreadingController("Apache Streams [default]", NUM_PROCESSORS, NUM_PROCESSORS * 5, Thread.NORM_PRIORITY);
-    private static final ThreadingController INSTANCE_HIGH_PRIORITY = new ThreadingController("Apache Streams [high]", NUM_PROCESSORS, NUM_PROCESSORS * 7, Thread.NORM_PRIORITY + 2);
+    private static ThreadingController INSTANCE_LOW_PRIORITY;
```

Comment:
Technically java convention states that only primiatives be all upper-case. I don't follow Java conventions for personal readability. I can change if you really really care.